compiled_starters/java/src/main/java/Main.java

```diff
@@ -1,12 +1,11 @@
-// Uncomment this block to pass the first stage
-// import java.util.Scanner;
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) throws Exception {
         // Uncomment this block to pass the first stage
         // System.out.print("$ ");
-        //
-        // Scanner scanner = new Scanner(System.in);
-        // String input = scanner.nextLine();
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
@glokta1 is this change intentional? Don't think this was uncommented earlier

Reply:
@rohitpaulk Yes. It wasn't consistent with other challenges – to pass first stage, they only need to uncomment the line that prints the `$ ` prompt.